globus_cli/commands/bookmark/create.py

```diff
@@ -7,11 +7,7 @@
 
 @click.command("create", help="Create a bookmark for the current user")
 @common_options
-@click.argument(
-    "endpoint_plus_path",
-    metavar=ENDPOINT_PLUS_REQPATH.metavar,
-    type=ENDPOINT_PLUS_REQPATH,
-)
```

Comment:
Should we amend or remove this comment altogether?: https://github.com/globus/globus-cli/blob/336675ff24da64c5ee487243f39ae39fc49a7e14/globus_cli/parsing/endpoint_plus_path.py#L29

It seems like the default metavar handling has been added but this property still needs to remain because the metavar varies based on usage so the comment is no longer accurate.

Reply:
Yeah, we still need the property, but the comment needs an update. I can try to remember to do it tomorrow morning.